java/pom.xml

```diff
@@ -197,11 +197,11 @@
     </profiles>
 
     <scm>
-        <url>https://github.com/envoyproxy/protoc-gen-validate</url>
-        <connection>scm:git:git@github.com:lyft/protoc-gen-validate.git</connection>
-        <developerConnection>scm:git:git@github.com:lyft/protoc-gen-validate.git</developerConnection>
-      <tag>HEAD</tag>
-  </scm>
+        <url>https://github.com/bufbuild/protoc-gen-validate</url>
+        <connection>scm:git:git@github.com:bufbuild/protoc-gen-validate.git</connection>
+        <developerConnection>scm:git:git@github.com:bufbuild/protoc-gen-validate.git</developerConnection>
+        <tag>HEAD</tag>
```

Comment:
This shouldn't really have any effect other than reducing the need to follow redirects.